transport/server.cc

```diff
@@ -755,6 +755,7 @@ future<fragmented_temporary_buffer> cql_server::connection::read_and_decompress_
 
 future<std::unique_ptr<cql_server::response>> cql_server::connection::process_startup(uint16_t stream, request_reader in, service::client_state& client_state,
         tracing::trace_state_ptr trace_state) {
+    ++_server._query_processor.local().get_cql_stats().startups;
```

Comment:
I wish there was a simpler way to get to these stats aside from going through a sharded query processor and its local instance. But I guess it's out of scope for this patch. No action needed.

Reply:
There is a way, store the stats in transport/ instead of query_processor. query_processor is in the cql layer, it doesn't know about STARTUP messages.

(yes, I'm allergic to stat updates requiring more than a single instruction)

Reply:
If we wanna move the stats from `query_processor` to transport this should be done in a separate issue (I can make one).
As for this one, I don't see what I could do - if I wanted to store a reference, I would have to store one for each shard if I understand correctly.

Reply:
We don't need to move the existing query_processor metrics. We only need to move the new per-message-type metrics.

Reply:
Alright
It sounds like it can be aggregated with
```cpp
uint64_t _connects = 0;
uint64_t _connections = 0;
uint64_t _requests_served = 0;
uint32_t _requests_serving = 0;
uint64_t _requests_blocked_memory = 0;
uint64_t _requests_shed = 0;
```
into a separate transport_stats object, or something alike. Will do so then